lib/plugins/static.js

```diff
@@ -51,6 +51,19 @@ function serveStatic(options) {
             return;
         }
 
+        if(req.headers['if-modified-since'])
+        {
+            var fromBrowser = new Date(req.headers['if-modified-since']);
+            var fromDrive = new Date(stats.mtime);
+            fromDrive.setMilliseconds(0);
```

Comment:
Does modify time in linux return milliseconds?? I haven't seen trouble without this.